base_classes/NXdata.nxdl.xml

```diff
@@ -64,12 +64,16 @@
 	</attribute>
 	<attribute name="signal">
 		<doc>
+			.. index:: find the default plottable data
 			.. index:: plotting
+			.. index:: signal attribute value
 			
-			Declares which dataset is the default. 
-			The value is the name of the dataset to be plotted.
-			A field of this name *must* exist (either as dataset 
-			or as a link to a dataset).
+			Declares which NeXus field (HDF5 dataset) is the default. 
+			The value is the name of the NeXus field to be plotted.
+			A field of this name *must* exist (either as a NeXus field 
+			or as a link to a NeXus field).
+			The value must match the :ref:`regular expression pattern for NXDL
+			group and field names &lt;validItemName>`.
```

Comment:
I think that this is the wrong place to be stating the requirement for a valid name and recommend removing the line.

Reply:
resolved below

Reply:
done in 4c5d50e